README.md

```diff
@@ -1,44 +1,27 @@
-# Standup
```

Comment:
lol yeah I don't know why I called this thing `standup`. My brain just disconnected from what it actually was.